packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -465,6 +465,13 @@ class WKWebView {
     throw UnimplementedError();
   }
 
+  /// An estimate of what fraction of the current navigation has been loaded.
+  ///
+  /// This value ranges from 0.0 to 1.0.
+  Future<double> get estimatedProgress {
```

Comment:
I missed this for earlier reviews (like `url` above), but this violates [Flutter style](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#getters-feel-faster-than-methods). We should make this `getEstimateProgress()`, and fix anything else that was using this pattern.

Reply:
I chose this style to make the Observe Value Api easier to use. For example, this PR includes the code

```dart
webView.addObserver(
        webView,
        keyPath: 'estimatedProgress',
        options: <NSKeyValueObservingOptions>{
          NSKeyValueObservingOptions.newValue,
        },
      );
```

If this or any other getters were changed to `getValueName`, this would not be as clear. However, that also reminds me that `url` translates to `WKWebView.URL`. So, I guess I have already changed field names to follow Flutter style. 

I'm fine with changing them, but I will also include the true name in the documentation so the future observing of fields doesn't require googling the class.